src/stdlib/builtin/lv_string_builtin.c

```diff
@@ -191,6 +191,14 @@ size_t lv_strlen(const char * str)
     return i;
 }
 
+size_t lv_strnlen(const char * str, size_t max_len)
+{
```

Comment:
Could you track for NULL here, it can be an assert or log it will make the debug easy if the things here get wrong

Reply:
Updated my commit with that.